external/external_custom/build.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<project name="External_custom Test" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build external_custom Docker image
+	</description>
+	<import file="${TEST_ROOT}/external/build.xml"/>
+
+	<!-- set properties for this build -->
+	<property name="TEST" value="derby" />
```

Comment:
derby needs to be changed to external_custom

Reply:
Yeah I will change it and try again

Reply:
will need something like: 
```
<property environment="env" />
<property name="EXT_CUSTOM_TARGET" value=${env.CUSTOM_EXTERNAL_TARGET}
```

Reply:
I have used
```
<property environment="env" />
<property name="EXTERNAL_CUSTOM_TARGET" value="${env.EXTERNAL_CUSTOM_TARGET}" />
``` 
but the build fails. 

For reading the test.properties file we can follow one of the following methods in the common_function
```while IFS= read -r line
do
    echo $line
done < external_custom/test.properties
```
or
```
cat external_custom/test.properties | while read line
do 
    echo $line
done
```

Reply:
For reading the test.properties file we can follow one of the following methods in the common_function
```
while IFS= read -r line
do
    echo $line
done < external_custom/test.properties
```
or
```
cat external_custom/test.properties | while read line
do 
    echo $line
done
```